src/main/java/com/projecturanus/betterp2p/util/GlStateManager.java

```diff
@@ -608,23 +606,23 @@ public static void glNormalPointer(int type, int stride, ByteBuffer buffer) {
     }
 
     public static void glTexCoordPointer(int size, int type, int stride, int buffer_offset) {
-        GL11.glTexCoordPointer(size, type, stride, (long) buffer_offset);
+        GL11.glTexCoordPointer(size, type, stride, buffer_offset);
```

Comment:
The long cast here is likely to indicate that the argument takes a long, not an int. While IntelliJ might not like it, it is common in code bases for clarity

Reply:
Thank you! I addressed it in https://github.com/GTNewHorizons/BetterP2P/pull/30/commits/bbd49d9638db031c71e2fa1d577d2baae0a48969